api/app/graphql/types/base_object.rb

```diff
@@ -3,5 +3,40 @@ class BaseObject < GraphQL::Schema::Object
     edge_type_class(Types::BaseEdge)
     connection_type_class(Types::BaseConnection)
     field_class Types::BaseField
+
+    def self.auto_field(*field_names)
+      field_names.each do |field_name|
+        raise("#{graphql_name} is not a model :(") unless ActiveRecord::Base.in?(klass.ancestors)
```

Comment:
This is just a sanity check. I want to fail fast here with a good error message if we try to `constantize` anything other than an ActiveRecord model.